R/write.magpie.R

```diff
@@ -237,7 +236,7 @@ write.magpie <- function(x, file_name, file_folder = "", file_type = NULL, appen
                           compression = 9, zname = "Time", zunit = zunit, varname = varnames[1], varunit = unit, ...)
       nc <- ncdf4::nc_open(filePath, write = TRUE)
       if (zunit == "years") {
-        ncdf4::ncvar_put(nc, "Time", years)
+        try(ncdf4::ncvar_put(nc, "Time", years), silent = TRUE)
```

Comment:
Why silent = TRUE?

Reply:
because the function also returns info that it failed even with silent = TRUE, with silent =FALSE it only adds a (in that case) rather misleading error message